gobblin-core/src/main/java/gobblin/source/extractor/extract/sftp/SftpLightWeightFileSystem.java

```diff
@@ -288,4 +288,13 @@ private static void safeDisconnect(Channel channel) {
       channel.disconnect();
     }
   }
+
+  /**
+   * Get the path as a string without schema or authority.
+   *
+   * E.g. Converts sftp://user/data/file.txt to /user/data/file.txt
+   */
+  private static String toUriPath(Path path) {
```

Comment:
This method seems useful for other classes as well, can you add it to either `PathUtils` or `HadoopUtils`